packages/handleRichTextBoxKeyEventNavigation/handleRichTextBoxKeyEventNavigation.js

```diff
@@ -3,18 +3,45 @@
 const TAB = 9;
 const ESCAPE = 27;
 
-export default function(e, record) {
+const TABBABLE_ELEMENT_SELECTOR = [
+    "a[href]",
+    "button",
+    "[contenteditable]:not([contenteditable='false'])",
+    "input",
+    "select",
+    "textarea",
+]
+    .map(
+        tagName =>
+            tagName + ":not([disabled]):not([tabindex^='-']):not(.a11y-skip)"
+    )
+    .concat(["[tabindex='0']:not([class*='a11y-hidden'])"])
+    .join(", ");
+
+// tabToAdjacentFocusableElement is a boolean. It allows to tab to the adjacent
```

Comment:
the name of the variable renders this comment redundant :)

Reply:
also, if the adjacent element is focusable, when do we NOT want to be able to tab-navigate to it? do we need this parameter?

Reply:
Consider this is a public package and people are using it, I don't want to introduce a big change to the current behavior if they are happy with what they have. But they can enable it if they want to. What do you think?

Reply:
I was hesitant considering this is a public package and people are using it, I didn't want to introduce a big change to the current behavior if they are happy with what they have. But I guess we can let the package versioning take care of it.  I will address it.

Reply:
package versioning to gate API behavior changes is fine, I just wanted to ensure we know we've got the correct behavior here